.github/workflows/copy-site-to-sourceforge.yml

```diff
@@ -6,17 +6,22 @@ name: "R: Copy Site to SourceForge"
 on:
   workflow_dispatch:
     inputs:
-      release-version:
+      version:
```

Comment:
sounds like not related optimization, move to separate PR to merge sooner.

Reply:
I'd say it is related too. We want to have the same release input name for both `workflow_dispatch` and `workflow_call` so we can target both with `${{ inputs.version }}`. Hence the change.

Reply:
Moved.

Reply:
This one can be resolved.